apps/utils/locust_tests/common/stats/aggregated_stats.py

```diff
@@ -35,8 +34,7 @@ def __init__(
             locust_env (Environment): Current Locust environment
             stats_tag (str): A string which tags the output JSON; used to distinguish between
             separate test runs
-            running_env (StatsEnvironment): A StatsEnvironment enum which represents the
-            current testing environment; either TEST, PROD or PROD_SBX
+            running_env (str): A string which represents the current testing environment
```

Comment:
Unless those strings are no longer valid, it'd probably be useful to keep the env strings and just also mention the new type of value that would be accepted. If its been entirely replaced, it'd still be nice to know what an example of the expected input format is.

Reply:
That's fair and I don't disagree - though I wanted to move references to environment logic out of locust since that's not something it necessarily needs to be aware of (and also why I didn't implement any validation). What do you think?

Reply:
Or rather...that's the direction I think we should be moving in.

Reply:
Moving it out makes sense if this will only ever be populated by automation; if it will ever be populated manually though it'd be really handy to know at least the format of how the string is expected to look.

Reply:
Updated :)